frontend/src/metabase/admin/databases/containers/DriverDeprecationBanner/DriverDeprecationBanner.jsx

```diff
@@ -0,0 +1,23 @@
+import { connect } from "react-redux";
+import _ from "underscore";
+import { isDeprecatedEngine } from "metabase/lib/engine";
+import Databases from "metabase/entities/databases";
+import DriverDeprecationBanner from "../../components/DriverDeprecationBanner";
+import { closeDeprecationNotice } from "../../database";
+import { isDeprecationNoticeEnabled } from "../../selectors";
+
+const mapStateToProps = (state, { databases = [] }) => ({
+  database: databases.find(database => isDeprecatedEngine(database.engine)),
```

Comment:
Since we're deprecating two drivers, it would be fair to show a banner containing both. Otherwise, users may use two deprecated drivers, notice the banner, close it, migrate from only one deprecated driver and miss the second one. Wdyt?

Reply:
Asked a question in the product doc. @jeff303 fyi

Reply:
As I recall, the idea here was that the listing page will already show you which ones are deprecated (since that word actually appears in the name, which is part of the list view).  And that would be sufficient to handle this scenario.  But that understanding was from a while ago.

Reply:
It sounds to me that we should just open the list view when clicking on `Show me` link. WDYT?

Reply:
Check with Bruno, but I think navigating to the list view is OK.  That way, they can immediately see which ones are relevant (since they will mention "deprecated" in the driver name on the list page).

Reply:
@alxnddr fixed the link